content/developers/developer-patterns/navigating-merklelogs/index.md

```diff
@@ -142,9 +143,9 @@ The structure of the header field is:
 
 The idtimestamp of the last leaf entry added to the log is always set in the header field.
 
-You can see from the hex data above, that the idtimestamp of the last entry in the log is `8e84dbbb6513a6`, the version is 0, the timestamp epoch is 1, the massif height is 14, and the massif index is 0.
+In the hex data above, the idtimestamp of the last entry in the log is `8e84dbbb6513a6`, the version is `0`, the timestamp epoch is `1`, the massif height is `14`, and the massif index is `0`.
```

Comment:
@robinbryce, should `8e84dbbb6513a6` be visible in [line 131](https://github.com/datatrails/datatrails-docs/pull/876/files#diff-cd9ba8ebeb5306314bf53c68a009c107b6699df01c0d37a89e52b93233296517R131)

Reply:
The diff is a little confusing, but `8e84dbbb6513a6` should be visible in the example output from the curl with the range header `-H "Range: bytes=0-31"`. If you've tried that comand and not obtained the original example

00000000000000008e84dbbb6513a60000000000000000000000010e00000000

Then something is wrong. uploading the .log file to https://hexed.it/ and have a look

Reply:
We'll fix once merged into your branch. This fork is making this complex to review.